README.md

```diff
@@ -15,13 +15,15 @@ I'm looking for a full remote position (EU)! See [edoardottt.com/cv](https://www
 
 | Company | Areas hiring remote | Full/Regional remote | Job Postings | Open for contractors |
 |---|---|---|---|---|
+| 1Password | All | Full remote | [Link](https://1password.com/careers) | ? |
 | Abnormal Security | Most jobs | Full remote okay | [Link](https://careers.abnormalsecurity.com/open-roles) | ? |
 | AirBnb | Some | Full remote okay | [Link](https://careers.airbnb.com/) | ? |
 | AirHouse | All | Full remote | [Link](https://jobs.lever.co/airhouse) | ? |
 | Alation | All | Full remote depending on the position | [Link](https://www.alation.com/careers/) | ? |
 | Alienvault | Some jobs | Full remote okay | [Link](http://jobs.jobvite.com/alienvault/) | ? |
 | Angle Health | All | Full remote | [Link](https://jobs.lever.co/AngleHealth) | ? |
 | Appsecconsulting | Some jobs | Full remote okay | [Link](https://www.appsecconsulting.com/company/careers/) | ? |
+| Arctic Wolf | Some | Full remote | [Link](https://arcticwolf.com/company/careers/) | ? |
```

Comment:
Are you sure this is full-remote?

Reply:
They do have a few fully-remote security positions posted there, maybe these would be consider "Regional remote" as they are cited with specific countries? Examples here: https://arcticwolf.wd1.myworkdayjobs.com/en-US/External/details/Senior-Berater-fr-Cybersecurity--m-w-d-_R24_365?jobFamilyGroup=e3a7c80f96c110014c11fae52d250000&locations=f6cfbd603ca11001f623f08595430000&locations=f6cfbd603ca11001f62459fe37ad0000&locations=f6cfbd603ca11001f6248a5f81c80000&locations=f6cfbd603ca11001f62492d1e1ab0000&locations=f6cfbd603ca11001f6249e3d8f5a0000

Reply:
yes, maybe including `regional` could work. Like `Full remote (Regional)`.